_pathways/path-interactive.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting and Using Interactive Sessions
+last_modified_at: 2024-05-29
+primary_reviewers: atombaby
+---
+
+This pathway is a step-by-step introduction to interactive sessions in the Rhino/Gizmo compute environment.
+
+## Pre-requisites
+
+ - Basic Unix/Linux commands `cd`, `ls`, and `pwd`
+ - Connection to Campus Network (VPN or on-campus)
+ - A terminal emulator ( _Terminal_ on Mac, _Putty_, or _WSL_ on Windows)
+ - {% glossary Node %}
+ - {% glossary Shell %}
+ - {% glossary Prompt %}
+ - {% glossary Session %}
+ - {% glossary Slurm %}
+ - {% glossary HutchNet ID %}
+ - {% glossary rhino %}
+ - {% glossary gizmo %}
+
+## Steps
+
+### Get a Hutchnet ID
+
+In order to use Fred Hutch batch computing resources you must have valid Fred Hutch credentials.  Sepcicially a [HutchNet ID](/scicomputing/access_credentials/#hutchnet-id).
+
+This HutchNet ID needs to be associated with a [PI account](/scicomputing/access_credentials/#accessing-slurm-clusters) in order to submit jobs.
+
+### Connect to the Campus Network
+
+Batch computing resources require that your local computer be connected to the campus network.  This authenticaiton process happens differently when you're on vs off campus.
+
+[When on campus](https://centernet.fredhutch.org/cn/u/center-it/help-desk/connecting-to-wifi.html)
+
+[When off campus](https://centernet.fredhutch.org/cn/u/center-it/help-desk/vpn.html)
+
+### Start a Terminal
+
+A terminal provides a text-based interface to computers (the "command line").
+
+Apple OSX has a built in terminal application. It can be found in _Applications->Utilities->Terminal_.  Other options are available that are more full featured such as [iterm2](https://iterm2.com/), which is one free option (please consider [donating](https://iterm2.com/donate.html)!)
+
+Windows has a few different terminals built in, but many of these are unsuitable for accessing Linux systems.  Two easy options are the Microsoft Terminal app (available in the App Store) and [PuTTY](/scicomputing/access_methods/#windows).
+
+### Shell Connection to the 'rhino' Login Nodes
+
+Once you have a connection to the Hutch network and a suitable terminal, the next step is to [connect to the login nodes known as `rhino`.](/scicomputing/access_methods/#ssh-connections)
```

Comment:
Can we make this link to just "/sciomputing/access_methods/" and leave off the "#ssh-connections"? The utility of that article makes more sense from the top rather than midway, just don't want to confuse new users.

Reply:
done